quickstarts/ruby/sidekiq/config.yml

```
@@ -5,9 +5,16 @@ description: |
 
   Simple, efficient background processing for Ruby. Sidekiq uses threads to
   handle many jobs at the same time in the same process.
+  
+  ### Sidekiq use cases
+  
+  Developers using Ruby on Rails to build applications often have other tasks needed to be done asynchronously such as data processing, batch executions, etc. Sidekiq is the answer to this need being a framework that provides background processing for Ruby apps. With this, developers can process jobs in the background and improve efficiency. Sidekiq leverages Redis as an in-memory data structure and it can be used with Resque, also a Redis-based job scheduler.
 
-
-  ## Get started!
+  ### New Relic + Sidekiq
+  
+  New Relic Ruby agent empowers you with a seamless monitoring of Sidekiq. Our dashboard helps you track key indicators like average CPU utilization and average physical memory. Also, New Relic’s pre-defined alert conditions notify you on performance metrics like Apdex score, memory usage, and transaction errors.
+ 
+ ## Get started!
```

Looks like this line isn't correctly formatted!
```suggestion
  ## Get started!
```